src/lib/pseudo.js

```diff
@@ -1,27 +1,45 @@
 'use strict';
 
-// Recursively walk an AST node searching for content leaves
-export function walkContent(node, fn) {
-  if (typeof node === 'string') {
-    return fn(node);
+// Walk an entry node searching for content leaves
+export function walkEntry(entry, fn) {
```

Comment:
the problem with the previous one was that:
```
{
  value: value
  attrs: attrs
  index: index
}
```

looks exactly the same as:

```
{
  one: 'one',
  two: 'two',
  other: 'other'
}
```

I need to separate walking entries from walking values.